src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a href="index.html">
+          <img
+            src="images/logo.png"
+            alt="logo_moyo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_item">
+            <a
+              href="index.html"
+              class="link is-active"
+            >
+              Apple
```

Comment:
Don't use spaces in the `href` attribute's value. If this is meant to be a placeholder for a navigation link, consider using `#` followed by the section name (e.g., `href="#apple"`).